webpack.config.js

```diff
@@ -39,7 +39,6 @@ const sassDocOpts = {
       templatePath: path.join(__dirname, 'templates'),
     },
     sass: {
-      implementation: 'sass',
```

Comment:
1) Dart Sass is the default
2) This needs to be an actual instance (e.g. `implementation: require('sass')`), not a string